docs/reference/module-api/module-overview.md

```diff
@@ -838,19 +838,82 @@ module.exports = {
 };
 ```
 
-### `queries(self)`
+### `queries(self, query)`
+
+The `queries` function is part of the Apostrophe API for querying the database. It takes two arguments: the module (`self`) and the query that will be called (`query`). It must return an object. That object can have two properties:
```

Comment:
"The query that will be called" is a little unsatisfying but, well, yes, and it avoids language like "the query that is being constructed."

Users are going to read this and bounce pretty hard, thinking "so... what? I call self.queries to make a query?"

But it's reference, not the guide you promise elsewhere, so.

Reply:
I don't hate "the query that is being constructed"

Reply:
Cool, I think that would be good language.